### DISCUSSION TITLE
What about some merge rules?

### DISCUSSION THREAD

**Discussion by mrhnrk** [tomorrow-one/transactional-outbox#147] [Ideas]
Hey 👋🏼 

I had a look into our merging rules and saw, that we didn't define anything there. Do we want to do so? 

My proposal would be:
- Require a pull request before merging
- Require status checks to pass before merging
- Require conversation resolution before merging

Maybe we can discuss here a bit to find a good fit for us 🤗

**Comment by magro** [comment]
I don't have a strict opinion on that, and can live with both the current status or with changes that you proposed.

**Reply by mrhnrk** [reply]
I'd like to set it up like I proposed to check if it is working for us. We can decide afterwards if we want to keep it or not, WDYT? 🤗

**Comment by mrhnrk** [comment]
I've updated the rules now 😊
